src/run-cache.ts

```
@@ -41,7 +44,41 @@ class RunCache {
+  /**
+   * Utility function to check if a key matches a pattern (supporting wildcards)
+   * @param pattern The pattern to match against (can include * wildcard)
+   * @param key The key to check
+   * @returns boolean indicating if the key matches the pattern
+   */
+  private static matchesPattern(pattern: string, key: string): boolean {
+    if (!pattern.includes("*")) {
+      return pattern === key;
+    }
+    
+    const regexPattern = new RegExp("^" + pattern.replace(/\*/g, ".*") + "$");
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)